CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 #### Added
 - Added support for `app-extension.intents-service` target type [#536](https://github.com/yonaskolb/XcodeGen/pull/536) @yonaskolb
+- Added support for custom framework root [#562](https://github.com/yonaskolb/XcodeGen/pull/562) @raptorxcz
```

Comment:
```suggestion
- Added support for custom `root` in `sdk` dependency [#562](https://github.com/yonaskolb/XcodeGen/pull/562) @raptorxcz
```

Reply:
ok done in 0533dda0a6966c2459b9a6881679d38f3be256c4